update supported OS & its version in UTokyo VPN (#962)
utelecon/utelecon.github.io: src/pages/en/utokyo_vpn/index.md

**Eligible user:** University of Tokyo faculties, staff and students having a UTokyo Account.

**Device Types and OS for VPN:** UTokyo VPN is accessible on the following versions of Windows and macOS. 
- Windows: Windows 10, 11
- macOS: macOS 11.x Big Sur, 12 Monterey, 13 Ventura, 14 Sonoma

> Moreover, iPhone, Android, and other smartphones can be used other than the PC, however, currently we provide limited English manuals or support for them. If you use it on these mobile devices, please keep the OS and dedicated app properly updated while using it.

Author: 日英で微妙に違うことが書いてあるのですよね。
もしかして、第一文のhoweverの前後を入れ替えた方がよいのですかね。

> Moreover, iPhone, Android, and other smartphones can be used other than the PC, however, currently we provide limited English manuals or support for them. If you use it on these mobile devices, please keep the OS and dedicated app properly updated while using it.

Author: ```suggestion
Although we provide limited English manuals or support for them currently, but you can use UTokyo VPN on iPhone, Android, and other smartphones other than PC. If you use it on these mobile devices, please keep the OS and dedicated app properly updated while using it.
```

Reviewer: ```suggestion
You can use UTokyo VPN on iPhone, Android, and other mobile devices, though we currently provide limited English manuals for them. If you use it on these mobile devices, please keep the OS and dedicated app properly updated while using it.
```
